<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CostAdjustment</class>
 <widget class="QWidget" name="CostAdjustment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox 
{  
background-color:white;
}
QPushButton:hover
{
background-color: rgb(180, 85, 100);
color:white;
}

QPushButton:hover:pressed
{
background-color:rgb(180, 85, 100);
color:white;
}

QPushButton
{
background-color:rgb(84, 115, 135);
color:white;
font-size:12px;
width: 100px;
border-radius:2px;
spacing:2px;
padding: 0px 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widgetControl" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <widget class="QLabel" name="labelState">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>1</y>
            <width>39</width>
            <height>39</height>
           </rect>
          </property>
          <property name="text">
           <string> 状态：</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBoxStateSelect">
          <property name="geometry">
           <rect>
            <x>46</x>
            <y>9</y>
            <width>39</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="labelDate">
          <property name="geometry">
           <rect>
            <x>91</x>
            <y>1</y>
            <width>39</width>
            <height>39</height>
           </rect>
          </property>
          <property name="text">
           <string> 调价日期：</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEditSelect">
          <property name="geometry">
           <rect>
            <x>136</x>
            <y>9</y>
            <width>39</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonSelect">
          <property name="geometry">
           <rect>
            <x>181</x>
            <y>12</y>
            <width>39</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/image/image/查询.png</normaloff>:/image/image/查询.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonSee">
          <property name="geometry">
           <rect>
            <x>226</x>
            <y>12</y>
            <width>39</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>成本价一览</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/image/image/预览.png</normaloff>:/image/image/预览.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="geometry">
           <rect>
            <x>271</x>
            <y>12</y>
            <width>38</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
          </property>
         </widget>
         <widget class="QLabel" name="labelSize">
          <property name="geometry">
           <rect>
            <x>315</x>
            <y>1</y>
            <width>16</width>
            <height>18</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetCost">
         <column>
          <property name="text">
           <string>调价单号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>调价原料</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>调价日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>状态</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidgetCost">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabAddCost">
        <attribute name="title">
         <string>添加成本调价单</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string> 基本信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>备注：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRemark">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>可不填</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string> 调价原料</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAddAdjustMat">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string> 添加调整原料...</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButtonSave">
             <property name="text">
              <string>  保存</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCancel">
             <property name="text">
              <string> 取消</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/image/image/cancel.png</normaloff>:/image/image/cancel.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabCostInfo">
        <attribute name="title">
         <string>成本调价单详情</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string> 基本信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>调价单号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNumberInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditStateInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>备注：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRemarkInfo">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="placeholderText">
              <string>可不填</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButtonLog">
           <property name="text">
            <string> 处理记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame
{
background-color:rgb(180, 85, 100);
color:white;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Aharoni</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string> 调价原料</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelTotalCount">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCommandLinkButton" name="commandLinkButtonEdit">
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: transparent;</string>
                </property>
                <property name="text">
                 <string> 编辑调价原料</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidgetCostInfo">
           <column>
            <property name="text">
             <string>原料名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>条形码</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>调价前成本价</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>调价后成本价</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>计价单位</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAddCostMatInfo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string> 添加调价原料</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonConfirmInfo">
           <property name="toolTip">
            <string>开始确认采购原料</string>
           </property>
           <property name="whatsThis">
            <string>开始确认采购原料</string>
           </property>
           <property name="text">
            <string> 确认调价</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/image/image/确认.png</normaloff>:/image/image/确认.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSubmitInfo">
           <property name="toolTip">
            <string>开始确认采购原料</string>
           </property>
           <property name="whatsThis">
            <string>开始确认采购原料</string>
           </property>
           <property name="text">
            <string> 提交</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/image/image/确认.png</normaloff>:/image/image/确认.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QPushButton" name="pushButtonSaveInfo">
             <property name="text">
              <string> 保存</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDeleteInfo">
             <property name="text">
              <string> 删除</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/image/image/删 除.png</normaloff>:/image/image/删 除.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCancelInfo">
             <property name="text">
              <string> 取消</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/image/image/cancel.png</normaloff>:/image/image/cancel.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
